File: core/src/main/java/com/fahamutech/kcore/admin/vholder/TestViewHolder.java
package com.fahamutech.kcore.admin.vholder;

import androidx.recyclerview.widget.RecyclerView;
import android.view.View;
import android.widget.ImageView;

import com.fahamutech.kcore.R;

public class TestViewHolder extends RecyclerView.ViewHolder {
    private ImageView imageView;

    public TestViewHolder(View itemView) {
        super(itemView);
        imageView = itemView.findViewById(R.id.testimony_image);
    }

    public ImageView getImageView() {
        return imageView;
    }
}

File: core/src/main/java/com/fahamutech/kcore/user/vholder/TestViewHolder.kt
package com.fahamutech.kcore.user.vholder

import android.view.View
import android.widget.ImageView
import androidx.recyclerview.widget.RecyclerView
import com.fahamutech.kcore.R

class TestViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
    val imageView: ImageView

    init {
        imageView = itemView.findViewById(R.id.testimony_image)
    }
}